posts/src/main/kotlin/pl/edu/pw/ia/posts/application/model/PostsRequests.kt

```diff
@@ -31,22 +35,28 @@ data class PostUpdateRequest(
 
 	@field:org.hibernate.validator.constraints.UUID(message = "Post id is invalid")
 	val postId: String,
+	@field:org.hibernate.validator.constraints.UUID(message = "Thread id is invalid")
+	val threadId: String,
 ) {
 	fun toCommand() =
 		UpdatePostCommand(
 			postId = UUID.fromString(postId),
 			content = content,
 			accountId = SecurityContext.getAccountId(),
+			threadId = UUID.fromString(threadId),
 		)
 }
 
 data class PostDeleteRequest(
 	@field:org.hibernate.validator.constraints.UUID(message = "Post id is invalid")
 	val postId: String,
+	@field:org.hibernate.validator.constraints.UUID(message = "Thread id is invalid")
```

Comment:
I think that posts will be identifiable by postid
Account idis needed to verify user